src/main/resources/js/analytics.js

```diff
@@ -0,0 +1,39 @@
+$(function() {  
+  //Google TagManager
+  var GTM_ID = '';
+  (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+  })(window,document,'script','dataLayer',GTM_ID);
+
+  var tgmIframe = '<noscript><iframe src="https://www.googletagmanager.com/ns.html?id='+GTM_ID+'"'+
+  ' height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>';
+  $(document.body).prepend(tgmIframe)
+
+  //DataLayer of GTM
```

Comment:
how about extract this part into a method instead of having a comment?

Reply:
@mwpeng Good, I will try Cheers